### ISSUE TITLE
[FeatureRequest] TestResult publish support 

### ISSUE THREAD

**Issue by SamJongenelen** [Augurk/Augurk.CommandLine#14]
I would like support for publishing test result files with Augurk (as this is possible with Pickles) :)

**Comment by jmezach** [comment]
Ah, yes, we used to have this feature and I think that the UI can still display it as well. I believe the main problem was that we couldn't figure out a way to map the executed tests to the associated feature after Visual Studio changed their output. But perhaps @marktaling can chime in on that as well.

That being said it is good to hear that there is still interest in this feature and we would love to bring it back. Unfortunately I can't provide a concrete timeline on when we will get to it. If you (or anyone else) has any suggestions on how to achieve this, that would be very welcome.

**Comment by SamJongenelen** [comment]
Ah, you mean, the specflow generated test names are hard to map back to the gherkin scenario names?
IIRC specflow keeps the original scenario names per generated test method. 
E.g. using Specflow xunit generation, the feature and scenario names are put in the testmethods' trait "testcategory".

**Comment by jmezach** [comment]
@SamJongenelen Yes, I believe that SpecFlow puts the associated feature and scenario names as attributes in the generated code. The problem is that these attributes do not show up in the test report file (.trx). The only thing that's there are the class and method names of the test that has executed.

Perhaps we can try to use reflection on the test assembly and get the attributes through there, but that means we need to know about the different test frameworks that are available. Additionally that would only work with .NET and we'd like to be as platform independent as we can.
